src/cascadia/TerminalApp/Jumplist.cpp

```diff
@@ -116,26 +112,22 @@ winrt::fire_and_forget Jumplist::UpdateJumplist(const CascadiaSettings& settings
 // - profiles - The profiles to add to the jumplist
 // Return Value:
 // - S_OK or HRESULT failure code.
-[[nodiscard]] HRESULT Jumplist::_updateProfiles(IObjectCollection* jumplistItems, winrt::Windows::Foundation::Collections::IVectorView<Profile> profiles) noexcept
+void Jumplist::_updateProfiles(IObjectCollection* jumplistItems, winrt::Windows::Foundation::Collections::IVectorView<Profile> profiles)
 {
-    try
-    {
-        for (const auto& profile : profiles)
-        {
-            // Craft the arguments following "wt.exe"
-            auto args = fmt::format(L"-p {}", to_hstring(profile.Guid()));
-
-            // Create the shell link object for the profile
-            winrt::com_ptr<IShellLinkW> shLink;
-            const auto normalizedIconPath{ _normalizeIconPath(profile.Icon()) };
-            RETURN_IF_FAILED(_createShellLink(profile.Name(), normalizedIconPath, args, shLink.put()));
+    // It's easier to clear the list and re-add everything. The settings aren't
+    // updated often, and there likely isn't a huge amount of items to add.
+    THROW_IF_FAILED(jumplistItems->Clear());

```

Comment:
This PR is much easier to review by [suppressing whitespace changes](https://github.com/microsoft/terminal/pull/13688/files?w=1). 🙂